kivymd/uix/button/button.py

```diff
@@ -903,333 +979,96 @@ def stop_elevation_anim(self):
         self._elevation = self.elevation
 
 
-class BaseCircularElevationButton(
-    RoundedRectangularElevationBehavior, BaseElevationButton
-):
-    pass
-
-
-class BaseRoundButton(RectangularRippleBehavior, BaseButton):
-    """
-    Base class for all round buttons, bringing in the appropriate
-    on-touch behavior
-    """
-
-    md_bg_color = ColorProperty([0.0, 0.0, 0.0, 0.0])
-
-    def __init__(self, **kwargs):
-        super().__init__(**kwargs)
-        Clock.schedule_once(self.set_text)
-        Clock.schedule_once(self.set_text_color)
-
-    def set_text_color(self, interval: Union[int, float]) -> NoReturn:
-        if not self.text_color:
-            self.text_color = self.theme_cls._get_text_color()
+# MD Button classes
 
-    def set_text(self, interval: Union[int, float]) -> NoReturn:
-        self.text = ""
 
-
-class BaseRectangleFlatButton(BaseFlatButton, BaseElevationButton):
-    line_width = NumericProperty(1)
-    """
-    Line width for button border.
-
-    :attr:`line_width` is an :class:`~kivy.properties.NumericProperty`
-    and defaults to `1`.
+class MDFlatButton(ButtonContentsText, BaseButton):
+    """A flat rectangular button with (by default) no border or
+    background. Text is the default text color.
     """
+    pass
 
-    line_color = ColorProperty(None)
-    """
-    Line color for button border.
 
-    :attr:`line_color` is an :class:`~kivy.properties.ColorProperty`
-    and defaults to `None`.
+class MDRaisedButton(
+    FakeRectangularElevationBehavior, ButtonElevationBehaviour,
+    ButtonContentsText, BaseButton
+):
+    """A flat button with (by default) a primary color fill and matching
+    color text.
     """
```

Comment:
"""
    A flat button with (by default) a primary color fill and matching
    color text.
    """